README.md

```diff
@@ -194,7 +194,9 @@ Additionally an [example AML application](/examples/aml/app.py) is provided in t
 
 ## Running the Examples
 
-The callback URL for both example projects will be `https://localhost:5000/yoti/auth`
+From the [Yoti Dashboard](https://www.yoti.com/dashboard/applications):
```

Comment:
Might be better as "*On* the Yoti Dashboard" or "*Use the Yoti Dashboard to...*"